content/threat-modelling.md

```diff
@@ -4,28 +4,28 @@ date: "2020-10-01"
 category: "Security Assurance"
 ---
 
-For organizations adopting cloud native, the primary mechanism of identifying risk and resulting controls and mitigations is to perform threat modeling of applications, data flows, and supporting processes and infrastructure. The method by which this is accomplished is minimally different from typical threat modeling. The below guidance is an enhancement of the four step OWASP threat modeling recommended for cloud native capabilities.
+For organizations adopting cloud native, a primary mechanism for identifying risks, controls and mitigations is to perform threat modeling. While there are many threat modeling techniques they share several core characteristics. All start with building a scoped representation of a system's architecture. This begins with identifying all important processes, data stores, and [security boundaries](https://csrc.nist.gov/glossary/term/Security_boundary). Once boundaries have been established and the relevant elements of the system partitioned within those boundaries, the next step is to model how these elements interact with special attention paid to any interactions that cross security boundaries. Tools like [OWASP Threat Dragon](https://owasp.org/www-project-threat-dragon/) and [Microsoft Threat Modeling Tool](https://docs.microsoft.com/en-gb/azure/security/develop/threat-modeling-tool) help guide users through this process and can perform analysis of the resulting diagrams to automatically detect some threats.
```

Comment:
Thinking about it, if you look at the text in : https://csrc.nist.gov/glossary/term/Security_boundary I don't think it's even remotely helpful in understanding what a security boundary is in this context.

Reply:
its a primary source definition, but i have to agree it lacks appropriate context here correcting with a better O'Reilly link (for sanity, we attempt to link to a source for defining content so we don't have to maintain the text)
```suggestion
For organizations adopting cloud native, a primary mechanism for identifying risks, controls and mitigations is to perform threat modeling. While there are many threat modeling techniques they share several core characteristics. All start with building a scoped representation of a system's architecture. This begins with identifying all important processes, data stores, and [security boundaries](https://www.oreilly.com/library/view/cissp-certified-information/9780470276884/9780470276884_security_boundaries.html#:~:text=A%20security%20boundary%20is%20the,a%20LAN%20and%20the%20Internet.). Once boundaries have been established and the relevant elements of the system partitioned within those boundaries, the next step is to model how these elements interact with special attention paid to any interactions that cross security boundaries. Tools like [OWASP Threat Dragon](https://owasp.org/www-project-threat-dragon/) and [Microsoft Threat Modeling Tool](https://docs.microsoft.com/en-gb/azure/security/develop/threat-modeling-tool) help guide users through this process and can perform analysis of the resulting diagrams to automatically detect some threats.
```